### PR TITLE
User account sign up Page

### PR DESCRIPTION
**Pull request by czhou578**
This is the basic design for the user account sign up page.

### CODE REVIEW

File: app/src/components/UI/Pages/Login.tsx [hunk 1]
```diff
@@ -24,11 +24,38 @@ export default function Login(): React.ReactElement {
   const navigate = useNavigate();
   const error = useAppSelector((state) => state.root.error);
 
+  useEffect(() => {
```
Comment (MakotoE) [inline app/src/components/UI/Pages/Login.tsx:27]: This is no longer required as it is done through `setCredentialsAndGetToken()`.

File: app/src/components/UI/Pages/Login.tsx [hunk 2]
```diff
@@ -24,11 +24,38 @@ export default function Login(): React.ReactElement {
...
+
+    generateNewToken();
+  }, []);
+
   const onSubmit = async () => {
     await setCredentialsMutation(credentials);
     navigate("/");
   };
 
+  const storeCredentials = () => {
```
Comment (MakotoE) [inline app/src/components/UI/Pages/Login.tsx:49]: Same with this.

**Review by MakotoE — CHANGES_REQUESTED** [review]

File: app/src/components/UI/Organisms/ReviewForm/ReviewForm.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import React, { useState } from "react";
+import { useState } from "react";
```
Comment (MakotoE) [inline app/src/components/UI/Organisms/ReviewForm/ReviewForm.tsx:1]: If the file uses JSX, `React` must be imported.

**Review by MakotoE — CHANGES_REQUESTED** [review]

**Comment by czhou578** [comment]
Can you look at the Signup.tsx page, if not already? Thanks!

**Comment by MakotoE** [comment]
I did. That looks good as is but I think it's missing the customer/vendor type selection.

**Comment by czhou578** [comment]
Wasn't that supposed to be for another separate page, as described in the SDD?

**Comment by MakotoE** [comment]
Yes. If you want to create a separate PR for that, that is fine too.

**Pull request closed without merging**